config/gitleaks.toml

```diff
@@ -167,6 +167,16 @@ title = "gitleaks config"
     description = "PyPI upload token"
     regex = '''pypi-AgEIcHlwaS5vcmc[A-Za-z0-9-_]{50,1000}'''
     tags = ["key", "pypi"]
+    
+[[rules]]
+    description = "Octopus deploy"
+    regex = '''api-AgEIcHlwaS5vcmc[A-Za-z0-9-_]{50,1000}'''
+    tags = ["key", "API"]
+    
+[[rules]]
+    description = "Sonarqube and other scanner tools"
+    regex = '''api-AgEIcHlwaS5vcmc[A-Za-z0-9-_]{50,1000}'''
+    tags = ["key", "api"]
```

Comment:
Why are the regex duplicated, shouldn't one be starting with `API` and the other with `api` ?
They only differ on the description and the tags.

Reply:
Apologies for typo,yes one is with API and another with api(not caps).With this it will detect API keys in lower and upper case format and make gitleaks much powerful.For now in our test using this template these keys never got detected ,so thought of improving this.Addition of this should fix the issue.

Will come up with one more issue in next merge request for autorization token.

should looms like this:
[[rules]]
    description = "Octopus deploy"
    regex = '''API-AgEIcHlwaS5vcmc[A-Za-z0-9-_]{50,1000}'''
    tags = ["key", "API"]

[[rules]]
    description = "Sonarqube and other scanner tools"
    regex = '''api-AgEIcHlwaS5vcmc[A-Za-z0-9-_]{50,1000}'''
    tags = ["key", "api"]

Reply:
Great! Can you provide another commit fixing the caps?

Thank you!

Reply:
Hi @w0rmr1d3r https://github.com/zricethezav/gitleaks/pull/643 is link for new pull request .It incluses API fixes as well as bearer token and jwt token detection regex expresessions

Reply:
Thank you @ds50421